plugin/cassandra-tables.json

```diff
@@ -5,7 +5,7 @@
       "deviceId": "text",
       "timestamp": "timestamp",
       "id": "int",
-      "parameters": "frozen<params>",
+      "parameters": "text",
```

Comment:
Is there anywhere a functionality which checks for supported types and UDT types, and throws an error if the user is mentioned unsupported type?

Reply:
There is no functionality like this, in case of wrong type it will try to execute query and fail on CQL level, hence throw an error directly from cassandra driver

Reply:
Error throwing is a bit not user friendly. From my perspective, would be better to create type checking functionality, and if it has decisioned that some type is wrong, make a call of PluginService onError function with pretty user friendly error message with rich explanation of how fix it.

Reply:
We already have handled this case (./plugin/CassandraPluginService.js:34). We catch promise exception and forward error to PluginService.onError